practice2_StarWarsDatabank/practice2_StarWarsDatabank/StarWarsViewController.swift

```diff
@@ -60,7 +58,7 @@ class StarWarsViewController: UIViewController {
         downoadTask.httpMethod = "GET"
         URLSession.shared.dataTask(with: downoadTask , completionHandler: {(data, response, error ) -> Void in
             if error != nil {
```

Comment:
it is better to use guard as a protection against errors
`guard error == nil else {`

Reply:
fixed